blocks/category-carousel-block/features/category-carousel/default.jsx

```diff
@@ -43,20 +53,61 @@ const customFieldGroups = Array(MAX_SLIDES)
 
 const mergeObjectArray = (accumulator, item) => ({ ...accumulator, ...item });
 
+const CategoryImage = ({ imageAlt, imageAuth, imageId, imageUrl }) => {
+	const imageAuthToken = useContent(
```

Comment:
@matthewroach : My only question on this PR is if we should be doing the signing in the blocks. These are images picked from PageBuilder so they SHOULD have the auth already?

Reply:
This is ok - this is primarily a fail safe for when an image is picked that may not have the auth token already generated.